app/[username]/(pages)/about/page.tsx

```diff
@@ -0,0 +1,12 @@
+export default async function UsernameAboutPage({
+  params,
+}: {
+  params: { username: string };
+}) {
+  const username = params.username.replaceAll("%40", "");
```

Comment:
I want to make the user profile page like a medium. In medium If you visit someone's profile, the pathname will look like this "medium.com/@Username". I also wanted to do like this so that it will be easy differentiate whether you are in user's profile or somewhere else. The reason of replacing is, there are articles and each article has its author. When a user click the author's username he will be thrown to the "/@Username" page and browser's path symbol "@" will be written as "%40" and it looks not good I this. Imagine"/%40Asliddin". That's why I replaced it.